forge-game/src/main/java/forge/game/cost/CostPartMana.java

```diff
@@ -132,6 +132,7 @@ public ManaCost getManaCostFor(SpellAbility sa) {
             return sa.getPaidList(CostExile.HashLKIListKey, true).get(0).getManaCost();
         }
         if (isEnchantedCreatureCost() && sa.getHostCard().isEnchantingCard()) {
+            // TODO human can still activate on TDFC backside
             return sa.getHostCard().getEnchantingCard().getManaCost();
```

Comment:
@tool4ever did you test what happen when it gets the "no cost" from a TDFC?

Reply:
For some reason it will allow payment, will debug it more in another PR